cypress/integration/workflow-termination.spec.js

```diff
@@ -49,6 +50,6 @@ describe('Terminate Button is Shown', () => {
     cy.wait('@settings-api');
     cy.wait('@workflow-api');
 
-    cy.get('[data-cy="terminate-button"]').should('not.exist');
+    cy.get('[aria-controls="workflow-actions"]').should('not.exist');
```

Comment:
I think it'd be worth it to add the modal part of the flow to these tests. Click button, assert modal is there with right title / confirm button text and input for reason for terminate.